relayer/relays/beacon/syncer/syncer.go

```diff
@@ -393,14 +411,14 @@ func IsInArray(values []uint64, toCheck uint64) bool {
 }
 
 func hexToBinaryString(rawHex string) string {
-	hex := strings.Replace(rawHex, "0x", "", -1)
```

Comment:
You can use `math/big` package to do this conversion for you. See https://go.dev/play/p/a3OICKkC9RN?v=goprev.go?download=true

Also instead of using `strings.Replace(rawHex, "0x", "", -1)` you could use `slices` with `rawHex[2:]`

Reply:
Addressed in [sync_committee_bits](https://github.com/Snowfork/snowbridge/pull/638/commits/8fdc1a156c72137a784def385d575baeb5483258).